vocabularies/language-iso639-1-english.ttl

```diff
@@ -0,0 +1,18 @@
+# English language only, subset of ISO639.
+
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+<http://www.w3.org/2004/02/skos/core#exactMatch> <http://id.loc.gov/vocabulary/iso639-2/eng>, <http://id.loc.gov/vocabulary/languages/eng> ;
+    <http://www.w3.org/2004/02/skos/core#inScheme> <http://id.loc.gov/vocabulary/iso639-1> ;
+    <http://www.w3.org/2004/02/skos/core#notation> "en" ;
+    <http://www.w3.org/2004/02/skos/core#prefLabel> "Englisch"@de, "English"@en, "anglais"@fr ;
+    <http://www.w3.org/2008/05/skos-xl#altLabel> "Englisch"@de, "English"@en, "anglais"@fr, [
+        a <http://www.w3.org/2008/05/skos-xl#Label> ;
+        <http://www.w3.org/2008/05/skos-xl#literalForm> "Englisch"@de
+    ], [
+        a <http://www.w3.org/2008/05/skos-xl#Label> ;
+        <http://www.w3.org/2008/05/skos-xl#literalForm> "anglais"@fr
+    ], [ 
+        a <http://www.w3.org/2008/05/skos-xl#Label> ;
+        <http://www.w3.org/2008/05/skos-xl#literalForm> "English"@en
+    ] .
```

Comment:
```suggestion
 @prefix lang: <http://linked.data.gov.au/def/language-iso639/> .

 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix prov: <https://www.w3.org/TR/prov-o/> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix sdo: <http://www.schema.org/> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core> .

<http://linked.data.gov.au/def/language-iso639> a owl:Ontology , skos:ConceptScheme ;
dct:created "2020-11-27"^^xsd:date ;
dct:creator <http://linked.data.gov.au/def/gsq-alias/des> ;
dct:modified "2020-11-27"^^xsd:date ;
dct:publisher <http://linked.data.gov.au/def/gsq-alias/gsq> ;
dct:source "IS0369 18:2018 standardized nomenclature to classify languages"@en ;
skos:definition "Languages used throughout the world"@en ;
skos:scopeNote "English language only, subset of ISO639."@en ;
skos:prefLabel "Languages"@en ;
rdfs:label   "Languages"@en ;
skos:hasTopConcept 
    lang:english .

<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
    sdo:name "Geological Survey of Queensland" ;
    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .

<http://linked.data.gov.au/org/des>  
    a sdo:Organization ;
    sdo:name "Department of Environment and Science"@en .

lang:english
    skos:exactMatch <http://id.loc.gov/vocabulary/iso639-2/eng>, <http://id.loc.gov/vocabulary/languages/eng> ;
    skos:inScheme <http://linked.data.gov.au/def/language-iso639> ;
    skos:notation "en" ;
    skos:prefLabel "Englisch"@de, "English"@en, "anglais"@fr .
```

Technically concepts Schemes should always have more than one concept. But I see where this is going. I'm not sure if there is a better way to link directly to the existing vocabularies. I _think_ the next edition of VocPrez is supposed to allow multi-source libraries.

Reply:
Is the alternative to pull in the entire language vocabulary and create a collection that contains English? 

Full vocab list sourced from https://id.loc.gov/vocabulary/iso639-1.html.

Reply:
Vocprez has a configuration file that you can add external vocabularies to and they will appear in VocPrez.
No need to copy.

Reply:
Documentation about this is here: https://rdflib.dev/VocPrez/#/README?id=data-sources

Reply:
@DavidCrosswellGSQ should we then just add the ISO639 as an external vocab source and then create a collection to limit to english.
We can expand the language if we need to include datasets from other languages in the future.

Reply:
@broughd-qld - best approach is to add the full ISO639 vocab.
If you want your drop-down or input field to only show English, then put that in your SPARQL query as a WHERE clause.

Reply:
@DavidCrosswellGSQ What is the best way to request the vocab gets loaded?

Reply:
@broughd-qld we can add them into the VocPrez configuration file as per https://rdflib.dev/VocPrez/#/README?id=data-sources

Reply:
@DavidCrosswellGSQ I've been unable to source a raw RDF file for the vocabulary from https://id.loc.gov/vocabulary/iso639-1.html. They all appear to be zip files. There also isn't a SPARQL endpoint on that service.
VocPrez help doesn't mention a zipped file. 
Are we best to copy the entire vocabulary RDF file from the source and load as per the normal procedure?

Reply:
The iso639 vocab is now live at https://vocabs.gsq.digital/vocabulary/iso639-1

I had to end up skosifying it and putting it at a linked.data.gov.au address.

There is still an issue with the Exact Match links not resolving, but I will need to take this up with Nick Car.

SPARQL query filtering by PrefLabel to get only English:
```sparql
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
SELECT ?con ?pl
WHERE {
   ?con a skos:Concept ;
       skos:prefLabel ?pl ;
       skos:inScheme <http://linked.data.gov.au/def/iso639-1> .
  FILTER(LANG(?pl)="en") .
  FILTER(?pl="English"@en) .
}
```

SPARQL query filtering by Concept:
```sparql
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
SELECT ?con ?pl
WHERE {
   ?con a skos:Concept ;
       skos:prefLabel ?pl ;
       skos:inScheme <http://linked.data.gov.au/def/iso639-1> .
  FILTER(LANG(?pl)="en") .
  FILTER(?con=<http://linked.data.gov.au/def/iso639-1/en>) .
}
```

An abbreviated query:
```sparql
PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
SELECT ?con ?pl
WHERE {
    ?con a skos:Concept ;
	skos:prefLabel ?pl ;
	skos:inScheme <http://linked.data.gov.au/def/iso639-1> .
  FILTER(LANG(?pl)="en" && ?pl="English"@en) .
}
```

Reply:
Thanks @DavidCrosswellGSQ